CHANGELOG.md

```diff
@@ -19,6 +19,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 - `mkKeyWalletFromFile` helper to use `cardano-cli`-style `skey`s.
 - Removed `Contract.Wallet.mkGeroWallet` and `Contract.Wallet.mkNamiWallet` - `Aff` versions should be used instead.
 - Added `getProtocolParameters` call to retrieve current protocol parameters from Ogmios (#541)
+- Updated `ogmios-datum-cache` to the recent revision
```

Comment:
Can you move this to the `### Changed` sub-heading below and remove the old note about ODC please?

I just noticed that all of these changes are in the wrong place (they should go under the different sub-headings)